lib/versioneye/crawlers/crates_crawler.rb

```diff
@@ -194,9 +197,11 @@ def self.upsert_product_owner(product_db, owner_doc, version_num)
       language: product_db[:language],
       prod_key: product_db[:prod_key],
       version: version_num,
-      name: owner_id
+      developer: owner_id
```

Comment:
@timgluz Good questions. I would say we have to add another field to the developers collection. A field for `fullname`.